LLM/deploy-llama3.1.md

```diff
@@ -11,9 +11,9 @@ Add these [policies](https://github.com/oracle-samples/oci-data-science-ai-sampl
 ```python
 # Install required python packages
 
-!pip install oracle-ads
-!pip install oci
-!pip install huggingface_hub
+pip install oracle-ads
```

Comment:
Should probably use -U to force latest ADS and other packages since this is such a new model all the libraries need to be newest